subsys/net/ip/net_if.c

```diff
@@ -175,6 +202,20 @@ static bool net_if_tx(struct net_if *iface, struct net_pkt *pkt)
 			context, status);
 
 		net_context_send_cb(context, status);
+
+#if defined(CONFIG_NET_CONTEXT_TIMESTAMP)
+		if (status >= 0 && start_timestamp.nanosecond &&
+		    curr_time > 0) {
+			/* So we know now how long the network packet was in
+			 * transit from where it was allocated to when we
```

Comment:
Minor: more logically would be to write "from when ... to when ..." (here "where" was used in the first case).